package/kedro_viz/models/graph.py

```diff
@@ -33,16 +33,23 @@
 import json
 import logging
 import re
+import os
 from dataclasses import InitVar, dataclass, field
 from enum import Enum
-from pathlib import Path
+from pathlib import Path, PurePosixPath
 from types import FunctionType
 from typing import Any, Dict, List, Optional, Set, Union, cast
+from datetime import datetime
+import pandas as pd
+from pandas.core.frame import DataFrame
+import plotly.express as px
+import plotly.io as pio 
 
 from kedro.io import AbstractDataSet
 from kedro.io.core import get_filepath_str
 from kedro.pipeline.node import Node as KedroNode
 from kedro.pipeline.pipeline import TRANSCODING_SEPARATOR, _strip_transcoding
+from kedro_viz.integrations.kedro import data_loader as kedro_data_loader

```

Comment:
Why rename the import? Is there another type of data loader we're using? I think since we're in Kedro it speaks for itself that the data loader loads Kedro data 🙂